notebooks/01-tessellate-ipu-tile-api-basics.ipynb

```diff
@@ -0,0 +1,590 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2d7b882e",
+   "metadata": {},
+   "source": [
+    "Copyright (c) 2023 Graphcore Ltd. All rights reserved."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "836513db",
+   "metadata": {},
+   "source": [
+    "# TessellateIPU - Basics of Tile Mapping on Tensors and Vertices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d9cf6f02",
+   "metadata": {},
+   "source": [
+    "The IPU is a highly parallel AI accelerator with 1,472 independent cores (also called IPU tiles) connected with an all-to-all IPU-exchange.\n",
+    "Each IPU tile has six independent program threads and 640 KB of local SRAM.\n",
+    "\n",
+    "**TessellateIPU** is a library exposing low-level IPU programming primitives in Python, allowing users to take full advantage of the IPU's unique architecture and features. In this tutorial notebook, we present the basics of the TessellateIPU API, showing how to:\n",
```

Comment:
Would be good here to add an expectation of what hardware this notebook requires and how long approx it might will take to run. Example in our notebook template: https://github.com/graphcore/examples-internal/blob/master/utils/templates/notebook_template.ipynb

Reply:
When you add the run on gradient button in the future could also be good to add a sentence saying this runs on paperspace - example in the template

Reply:
Thanks for the suggestion. I just added a short sentence saying that the notebook can be run on IPU hardware or IPU model, and every cell should only take a couple of seconds